src/Adapter/Language/LanguageDataProvider.php

```diff
@@ -46,4 +47,51 @@ public function getLanguages($active = true, $shopId = false, $onlyIds = false)
     {
         return Language::getLanguages($active, $shopId, $onlyIds);
     }
+
+    /**
+     * Returns language code by iso code
+     *
+     * @param string $isoCode - ISO 3166-2 alpha-2 format code
+     *
+     * @return false|null|string
+     */
+    public function getLanguageCodeByIso($isoCode)
+    {
+        return Language::getLanguageCodeByIso($isoCode);
+    }
+
+    /**
+     * Gets language details from json file
+     *
+     * @param string $locale
+     *
+     * @return bool|mixed
+     *
+     * @throws Exception
+     */
+    public function getJsonLanguageDetails($locale)
+    {
+        return Language::getJsonLanguageDetails($locale);
+    }
+
+    /**
+     * Gets the files list for given language
+     *
+     * @param $isoFrom
+     * @param $themeFrom
+     * @param bool $isoTo
+     * @param bool $themeTo
+     * @param bool $modules
+     *
+     * @return array
+     */
+    public function getFilesList(
+        $isoFrom,
+        $themeFrom,
+        $isoTo = false,
+        $themeTo = false,
+        $modules = false
```

Comment:
what happens if $modules equal true?

Can we make a function with only parameters for our current need: $isoFrom, $themeFrom, $isoTo = false, $themeTo = false, maybe getFilesListInModules if I have well understood what is done here?